apiExamples/dynamicMenu.js

```diff
@@ -0,0 +1,50 @@
+import { DynamicData } from './dynamicMenuDataApi';
```

Comment:
This new work exhibits the same issue that was reported in this issue

- https://github.com/AlaskaAirlines/auro-combobox/issues/68

I understood this work to illustrate that the problem was related to implementation, not the element itself. 

If there is an issue with the element, we need to resolve that before we can release this. 

![Nov-03-2022 11-32-48](https://user-images.githubusercontent.com/181089/199806741-7c7e14c3-33aa-4323-a10e-c72083bd4a53.gif)

Reply:
@blackfalcon Regardless of the rendering bug we have needed to get a working example of using combobox with a dynamic menu driven from an external API. This PR is only attempting to address the documentation, not the rendering issue.

We also still don't have a solid answer as to why the rendering issue is happening. I know @jordanjones243 intention here is to do the documentation himself, and then pass the baton off to me to figure out why the rendering issue occurs. 

There is no reason to not include the documentation about how to use dynamic menus before we figure out the rendering issue. This workflow is already in use in Flight Search.

Reply:
That's fine. I just want to make sure that we are not missing that part.